Tasks/DockerV2/dockerpush.ts

```diff
@@ -79,3 +99,58 @@ export function run(connection: ContainerConnection, outputUpdate: (data: string
 
     return promise;
 }
+
+async function publishToImageMetadataStore(connection: ContainerConnection, imageName: string, tags: string[], digest: string, dockerFilePath: string): Promise<any> {
```

Comment:
Move this method to pipelineutils.ts. https://github.com/Microsoft/azure-pipelines-tasks/blob/master/Tasks/Common/docker-common/pipelineutils.ts

Reply:
This method uses sendRequest method from utilities-common/restutilities. If I move this to docker-common/pipelineUtils, I get the build errors, saying that these two util directories cannot have dependency on each other.
So I am placing this code in dockertask itself

Reply:
In that case only move the part where you construct the requestBody